tests/all/volumes.t

```diff
@@ -111,3 +111,37 @@ nodes.each do |node|
   USE_NODE node
   puts TEST "cat /var/log/kadalu/mgr.log"
 end
+
```

Comment:
Add these tests after Line number 102 and before cleanup

Reply:
Done